integration/test/test_power_governor.py

```diff
@@ -114,11 +109,8 @@ def test_power_consumption(self):
             pkg_energy_cols = [s for s in power_data.keys() if 'ENERGY_PACKAGE' in s]
             dram_energy_cols = [s for s in power_data.keys() if 'ENERGY_DRAM' in s]
             power_data['SOCKET_POWER'] = power_data[pkg_energy_cols].sum(axis=1) / power_data['ELAPSED_TIME']
-            power_data['DRAM_POWER'] = power_data[dram_energy_cols].sum(axis=1) / power_data['ELAPSED_TIME']
-            power_data['COMBINED_POWER'] = power_data['SOCKET_POWER'] + power_data['DRAM_POWER']
 
             pandas.set_option('display.width', 100)
-            launcher.write_log(name, 'Power stats from {} :\n{}'.format(nn, power_data.describe()))
```

Comment:
I see you've removed all the write_log calls.  This data was useful to track power stats from these tests over time.  Is this being captured somewhere else now?

Reply:
I'm not sure how to support this feature and also support skip launch.  The log file is created when the launch happens.  These calls to "write_log" are happening in the the assertion part of the code.  We could just write this to standard output?

Reply:
I'm fine with either a new log file that's specific to this test (so you can just create/open/write to it here, and avoid ```launcher.write_log``` or blasting it all in to stdout.  If it's a separate file, it'll be easier to sift through later.